lib/hg-utils.coffee

```diff
@@ -291,7 +292,10 @@ class Repository
         if err
           reject err
         if stderr?.length > 0
-          reject stderr
+          if isWindows and stderr.length > windowsMaxPathLength
+            resolve(stdout)
```

Comment:
Is `'could not find file' in stderr` a better solution than `stderr.length > 260`?